src/Essentials/src/DeviceDisplay/DeviceDisplay.ios.cs

```diff
@@ -18,9 +18,13 @@ protected override DisplayInfo GetMainDisplayInfo()
 			var bounds = UIScreen.MainScreen.Bounds;
 			var scale = UIScreen.MainScreen.Scale;
 
+			// TODO: warning doesn't seem correct, file an issue?
+			// error CA1416: This call site is reachable on: 'iOS' 10.0 and later, 'maccatalyst' 10.0 and later. 'UIScreen.MaximumFramesPerSecond' is only supported on: 'ios' 10.3 and later, 'maccatalyst' 10.3 and later, 'tvos' 10.2 and later.
+#pragma warning disable CA1416
 			var rate = (OperatingSystem.IsIOSVersionAtLeast(10, 3) || OperatingSystem.IsTvOSVersionAtLeast(10, 3))
 				? UIScreen.MainScreen.MaximumFramesPerSecond
 				: 0;
+#pragma warning restore CA1416
```

Comment:
@rolfbjarne this doesn't quite feel right, I got the warning above building the net7.0-ios target framework. Did anything change with the `MaximumFramesPerSecond` API?

Or is it a bug in the analyzer that emits `CA1416`?

Reply:
Once I fixed a couple in `Core.csproj`, it just seems like a lot more come up now:
```
C:\src\maui\src\Controls\src\Core\Compatibility\Handlers\Shell\iOS\ShellTableViewController.cs(80,5): error CA1416: This call site is reachable on: 'iOS' 10
.0 and later, 'maccatalyst' 10.0 and later. 'UIScrollView.ContentInsetAdjustmentBehavior' is only supported on: 'ios' 11.0 and later, 'maccatalyst' 11.0 and
 later, 'tvos' 11.0 and later. [C:\src\maui\src\Controls\src\Core\Controls.Core.csproj::TargetFramework=net7.0-ios]
C:\src\maui\src\Controls\src\Core\Compatibility\Handlers\Shell\iOS\ShellSectionRootRenderer.cs(78,5): error CA1416: This call site is reachable on: 'iOS' 10
.0 and later, 'maccatalyst' 10.0 and later. 'UIView.InsetsLayoutMarginsFromSafeArea' is only supported on: 'ios' 11.0 and later, 'maccatalyst' 11.0 and late
r, 'tvos' 11.0 and later. [C:\src\maui\src\Controls\src\Core\Controls.Core.csproj::TargetFramework=net7.0-ios]
C:\src\maui\src\Controls\src\Core\Compatibility\Handlers\Shell\iOS\ShellScrollViewTracker.cs(73,21): error CA1416: This call site is reachable on: 'iOS' 10.
0 and later, 'maccatalyst' 10.0 and later. 'UIScrollView.AdjustedContentInset' is only supported on: 'ios' 11.0 and later, 'maccatalyst' 11.0 and later, 'tv
os' 11.0 and later. [C:\src\maui\src\Controls\src\Core\Controls.Core.csproj::TargetFramework=net7.0-ios]
C:\src\maui\src\Controls\src\Core\Compatibility\Handlers\Shell\iOS\ShellScrollViewTracker.cs(129,5): error CA1416: This call site is reachable on: 'iOS' 10.
0 and later, 'maccatalyst' 10.0 and later. 'UIScrollView.ContentInsetAdjustmentBehavior' is only supported on: 'ios' 11.0 and later, 'maccatalyst' 11.0 and
later, 'tvos' 11.0 and later. [C:\src\maui\src\Controls\src\Core\Controls.Core.csproj::TargetFramework=net7.0-ios]
C:\src\maui\src\Controls\src\Core\Compatibility\Handlers\Shell\iOS\ShellSectionRootRenderer.cs(511,109): error CA1416: This call site is reachable on: 'iOS'
 10.0 and later, 'maccatalyst' 10.0 and later. 'UIView.SafeAreaInsets' is only supported on: 'ios' 11.0 and later, 'maccatalyst' 11.0 and later, 'tvos' 11.0
 and later. [C:\src\maui\src\Controls\src\Core\Controls.Core.csproj::TargetFramework=net7.0-ios]
C:\src\maui\src\Controls\src\Core\Compatibility\Handlers\Shell\iOS\ShellSectionRootRenderer.cs(523,12): error CA1416: This call site is reachable on: 'iOS'
10.0 and later, 'maccatalyst' 10.0 and later. 'UIView.SafeAreaInsets' is only supported on: 'ios' 11.0 and later, 'maccatalyst' 11.0 and later, 'tvos' 11.0
and later. [C:\src\maui\src\Controls\src\Core\Controls.Core.csproj::TargetFramework=net7.0-ios]
C:\src\maui\src\Controls\src\Core\Compatibility\Handlers\Shell\iOS\ShellSectionRootRenderer.cs(524,11): error CA1416: This call site is reachable on: 'iOS'
10.0 and later, 'maccatalyst' 10.0 and later. 'UIView.SafeAreaInsets' is only supported on: 'ios' 11.0 and later, 'maccatalyst' 11.0 and later, 'tvos' 11.0
and later. [C:\src\maui\src\Controls\src\Core\Controls.Core.csproj::TargetFramework=net7.0-ios]
C:\src\maui\src\Controls\src\Core\Compatibility\Handlers\Shell\iOS\ShellSectionRootRenderer.cs(526,14): error CA1416: This call site is reachable on: 'iOS'
10.0 and later, 'maccatalyst' 10.0 and later. 'UIView.SafeAreaInsets' is only supported on: 'ios' 11.0 and later, 'maccatalyst' 11.0 and later, 'tvos' 11.0
and later. [C:\src\maui\src\Controls\src\Core\Controls.Core.csproj::TargetFramework=net7.0-ios]
C:\src\maui\src\Controls\src\Core\Compatibility\Handlers\Shell\iOS\ShellSectionRootRenderer.cs(525,13): error CA1416: This call site is reachable on: 'iOS'
10.0 and later, 'maccatalyst' 10.0 and later. 'UIView.SafeAreaInsets' is only supported on: 'ios' 11.0 and later, 'maccatalyst' 11.0 and later, 'tvos' 11.0
and later. [C:\src\maui\src\Controls\src\Core\Controls.Core.csproj::TargetFramework=net7.0-ios]
C:\src\maui\src\Controls\src\Core\Platform\GestureManager\GestureManager.iOS.cs(712,7): error CA1416: This call site is reachable on: 'iOS' 10.0 and later,
'maccatalyst' 10.0 and later. 'UIView.AccessibilityRespondsToUserInteraction' is only supported on: 'ios' 13.0 and later, 'maccatalyst' 13.0 and later, 'tvo
s' 13.0 and later. [C:\src\maui\src\Controls\src\Core\Controls.Core.csproj::TargetFramework=net7.0-ios]
C:\src\maui\src\Controls\src\Core\Platform\GestureManager\GestureManager.iOS.cs(553,55): error CA1416: This call site is reachable on: 'iOS' 10.0 and later,
 'maccatalyst' 10.0 and later. 'UIView.AccessibilityRespondsToUserInteraction' is only supported on: 'ios' 13.0 and later, 'maccatalyst' 13.0 and later, 'tv
os' 13.0 and later. [C:\src\maui\src\Controls\src\Core\Controls.Core.csproj::TargetFramework=net7.0-ios]
C:\src\maui\src\Controls\src\Core\Platform\GestureManager\GestureManager.iOS.cs(554,6): error CA1416: This call site is reachable on: 'iOS' 10.0 and later,
'maccatalyst' 10.0 and later. 'UIView.AccessibilityRespondsToUserInteraction' is only supported on: 'ios' 13.0 and later, 'maccatalyst' 13.0 and later, 'tvo
s' 13.0 and later. [C:\src\maui\src\Controls\src\Core\Controls.Core.csproj::TargetFramework=net7.0-ios]
C:\src\maui\src\Controls\src\Core\Handlers\Items\iOS\ItemsViewLayout.cs(53,5): error CA1416: This call site is reachable on: 'iOS' 10.0 and later, 'maccatal
yst' 10.0 and later. 'UICollectionViewFlowLayout.SectionInsetReference' is only supported on: 'ios' 11.0 and later, 'maccatalyst' 11.0 and later, 'tvos' 11.
0 and later. [C:\src\maui\src\Controls\src\Core\Controls.Core.csproj::TargetFramework=net7.0-ios]
C:\src\maui\src\Controls\src\Core\Compatibility\Handlers\ListView\iOS\ListViewRenderer.cs(233,7): error CA1416: This call site is reachable on: 'iOS' 10.0 a
nd later, 'maccatalyst' 10.0 and later. 'UITableView.SectionHeaderTopPadding' is only supported on: 'ios' 15.0 and later, 'maccatalyst' 15.0 and later, 'tvo
s' 15.0 and later. [C:\src\maui\src\Controls\src\Core\Controls.Core.csproj::TargetFramework=net7.0-ios]
C:\src\maui\src\Controls\src\Core\Handlers\Items\iOS\ItemsViewLayout.cs(568,23): error CA1416: This call site is reachable on: 'iOS' 10.0 and later, 'maccat
alyst' 10.0 and later. 'UIScrollView.AdjustedContentInset' is only supported on: 'ios' 11.0 and later, 'maccatalyst' 11.0 and later, 'tvos' 11.0 and later.
[C:\src\maui\src\Controls\src\Core\Controls.Core.csproj::TargetFramework=net7.0-ios]
C:\src\maui\src\Controls\src\Core\Handlers\Items\iOS\ItemsViewController.cs(147,5): error CA1416: This call site is reachable on: 'iOS' 10.0 and later, 'mac
catalyst' 10.0 and later. 'UIScrollView.ContentInsetAdjustmentBehavior' is only supported on: 'ios' 11.0 and later, 'maccatalyst' 11.0 and later, 'tvos' 11.
0 and later. [C:\src\maui\src\Controls\src\Core\Controls.Core.csproj::TargetFramework=net7.0-ios]
C:\src\maui\src\Controls\src\Core\Compatibility\Handlers\Shell\iOS\ShellFlyoutLayoutManager.cs(131,6): error CA1416: This call site is reachable on: 'iOS' 1
0.0 and later, 'maccatalyst' 10.0 and later. 'UIScrollView.ContentInsetAdjustmentBehavior' is only supported on: 'ios' 11.0 and later, 'maccatalyst' 11.0 an
d later, 'tvos' 11.0 and later. [C:\src\maui\src\Controls\src\Core\Controls.Core.csproj::TargetFramework=net7.0-ios]
```

Reply:
@jonathanpeppers did MAUI recently change their default min OS version? At the very least the warning about ContentInsetAdjustmentBehavior is correct (it's available on iOS 11+, and that hasn't changed)

If MAUI didn't change the min OS version, I think this is an analyzer bug.

Reply:
I filed https://github.com/dotnet/roslyn-analyzers/issues/6158